### DISCUSSION TITLE
iOS configuration yaml

### DISCUSSION THREAD

**Discussion by vromaniello** [stella-cv/stella_vslam#268] [Q&A] [answered]
Hello. I want to run OpenVSLAM on a video made with my iPhone's camera.

How can i create the yaml configuration file?
Such as https://github.com/OpenVSLAM-Community/openvslam/blob/main/example/aist/equirectangular.yaml.

I looked at the documentation, but I couldn't find how to make a yaml file.

Thank you for the support.

**Comment by ymd-stella** [comment, marked as answer]
Please refer to the configuration file for perspective projection and monocular.
See https://github.com/stella-cv/stella_vslam/blob/main/example/euroc/EuRoC_mono.yaml.
